src/api.ts

```diff
@@ -24,4 +24,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId: number | undefined) => get<User>(`/users/${userId}`);
```

Comment:
think the best way to use null for this case ( you don`t need to write undefined, it s the default empty value that gives js, as dev you can try to use others way, in this case - null.)

Reply:
yup, thanks